UniversalScannerService/Startup.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.ResponseCompression;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace UniversalScannerService
+{
+    public class Startup
+    {
+        private static readonly string AllowAnyOriginPolicy = "AllowAnyOriginPolicy";
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddCors(option =>
+            {
+                option.AddPolicy(AllowAnyOriginPolicy, policy =>
+                {
+                    policy.AllowAnyOrigin().AllowAnyHeader().AllowAnyMethod();
+                });
+            });
+
+            services.AddGrpc();

```

Comment:
What is the added value to use Grpc on a small tool like this? I do not see a real added value here.